lib/mastermind/display.rb

```diff
@@ -18,18 +18,18 @@ def display_welcome
       @output.puts(START_PROMPT)
     end
 
-    def guess_message(guess)
-      @output.puts(GUESS_MESSAGE+ guess)
+    def guess_message(guess, number_of_guesses)
+      @output.puts(number_of_guesses.to_s + ': ' + GUESS_MESSAGE + guess)
```

Comment:
It might be clearer to turn this into `"#{number_of_guesses}: #{GUESS_MESSAGE}#{guess}"`. My message formatting might be a little off though.